src/command_modules/azure-cli-container/azure/cli/command_modules/container/tests/latest/test_container_commands.py

```diff
@@ -125,7 +127,7 @@ def test_container_create_with_acr(self, resource_group, resource_group_location
         registry_username = 'clitestregistry1'
         registry_server = '{}.azurecr.io'.format(registry_username)
         image = '{}/nginx:latest'.format(registry_server)
-        password = 'EsYA8DyuSaNv+3HyGyHoiIIafktf0kE6'
+        password = 'rylgTHkXbBp6k/F0Gd3lOpVnGU0dVEq4'
```

Comment:
Out of curiosity, why did you change the password?

Reply:
Okay, I just read the comment. But please bear with me. I don't understand conceptually why the password has to be automatically re-generated every time the test is run live. Does this have to do with the fact that the test is created using the clitestregistry1 nginx image?